spynnaker/pyNN/models/common/neuron_recorder.py

```diff
@@ -971,7 +1086,7 @@ def get_variable_sdram_usage(self, vertex_slice):
                     fixed_sdram += (per_record - average_per_timestep)
         for variable in self.__per_timestep_recording:
             per_timestep_sdram += self.get_buffered_sdram_per_record(
-                variable, vertex_slice)
+                variable, vertex_slice, max_rewires_per_ts)
```

Comment:
Avoided by not having the variable in this loop - instead can be calculated separately below here